### DISCUSSION TITLE
bot with vpn

### DISCUSSION THREAD

**Discussion by Mellwins** [jagrosh/MusicBot#1745] [Questions]
can u guys help, how to setup bot with vpn or zapret like Goodbyedpi, country Russia, main problem is youtube slowed for russia and bot is cannot play from youtube

**Comment by virina4351-design** [comment]
https://github.com/jagrosh/MusicBot/discussions/1745#discussion-7387271

**Reply by virina4351-design** [reply]
https://youtu.be/5uh9imWsNNY?si=lL6BSEhJeqb_QrrD
